index.Rmd

```diff
@@ -37,7 +37,7 @@ _Please address all general course-related email communication to joel.ostblom@m
 Prof. Martin Krkosek, martin.krkosek@utoronto.ca, 416-978-3839, RW402
 
 ## Course Website and Blackboard
-All course information is accessible on its own website (https://uoftcoders.github.io/rcourse/) and will also be stored on Blackboard (https://portal.utoronto.ca/). You will have access to the Syllabus, Lecture Schedule & Recommended Readings for each lecture, Lab Schedule, Lab Instructions, Lecture slides (pdf files) and Lab Data sheets. If you have any problem accessing the material, let us know right away so we can fix the problem (e-mail is best).
+All course information is accessible [on its own website](https://uoftcoders.github.io/rcourse/) and will also be stored on [Blackboard](https://portal.utoronto.ca/). You will have access to the Syllabus, Lecture Schedule, and Lecture slides. If you have any problem accessing the material, let us know via email right away so we can fix the problem.
```

Comment:
Agreed. I also reformatted and removed "lecture schedule" due to redundancy with "syllabus", in my opinion.